README.md

```diff
@@ -2,20 +2,40 @@
 This section describes tools that help you with coding. We provide scripts that allows to
 clone all repositories and install artifacts to Maven local / Maven Central Snapshot repositories.
 
+## Prerequisites on Windows
+- Install and configure WSL - Windows Subsystem for Linux - it's really necessary, without it 
```

Comment:
Install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL) and configure:
- JDK 8
- Gradle 5+
- Maven

Reply:
fixed